### DISCUSSION TITLE
What's the answer, until today I have this doubt and I can't find the solution, someone for a minute, and try to answer?

### DISCUSSION THREAD

**Discussion by FranciscoNetoBr** [ant-media/Ant-Media-Server#5021] [Q&A]
I am unable to repost a stream, for example in the screenshots. I have read as many articles and posts as possible and have not found a solution.

VPS Settings - Oracle Cloud
Type: VM.Standard.A1.Flex
Number of OCPUs: 4
Network Bandwidth (Gbps): 4
Memory (GB): 24
Image: Canonical-Ubuntu-22.04-aarch64-2023.02.15-0

Ant Media simply doesn't play the URL (ex. rtmp://129.148.56.238/Youtube/Lz6b1W9LQq3d1677958482905)
--------------- If you have a solution, please provide a more detailed explanation. I am not an analyst or system developer.
[https://github.com/ant-media/Ant-Media-Server/issues/4971](url)
#https://github.com/ant-media/Ant-Media-Server/issues/4971

**Comment by Mohit-3196** [comment]
Hi @FranciscoNetoBr ,
How are you?

Sorry for the trouble that you had. Let me try to help you with this 

> Ant Media simply doesn't play the URL (ex. rtmp://129.148.56.238/Youtube/Lz6b1W9LQq3d1677958482905)

As I can see from the GitHub issue screenshot, it looks as if you are trying to pull this RTMP URL as a stream source from Ant Media Server. Is it so? or do you have some other use case here?
Also this RTMP URL rtmp://129.148.56.238/Youtube/Lz6b1W9LQq3d1677958482905 doesn't seem to be active. Please make sure that it is active.

Also in the second question, you are trying to pull a YouTube URL, due to security reasons and youtube permissions, you cannot pull it from YouTube and re-stream.
However, If it were your own stream source, you could simulcast it to YouTube Live and several other social media platforms
https://antmedia.io/docs/guides/publish-live-stream/simulcasting/

Thank you

**Reply by FranciscoNetoBr** [reply]
Let me start by thanking you for your interest in helping me.

You know I've been looking for a solution for a while. Thanks.

First Answer:

Yes, I was using the Ant Media Server RTMP URL source.
And when the test was performed, the source was active.

Second answer:

Yes, basically the sources I'm using are from Youtube (street and landscape cameras), free for retransmission.

That I would like to use them for Infinite Lives.

I manage to do this with OBS studio, however, OBS consumes a lot of my server, both in processing and memory, with constant transmission drops. Which makes my project unfeasible.

That's why I try to use Ant Media Server, which is light and reliable.

**Comment by Mohit-3196** [comment]
Hi @FranciscoNetoBr ,
Thank you for the answers.

For the RTMP stream source, RTMP pull is not maintained for AMS. We even do not recommend playing streams with RTMP and is also not available by default.
Hence RTMP stream can be pushed to AMS but pulling it is not maintained.

For restreaming the YouTube video as we discussed earlier, due to security reasons and youtube permissions, one cannot easily pull it from YouTube and re-stream.
OBS or other encoders are a workaround solution as you mentioned.
There are chances of doing it via ffmpeg but again, there's no guarantee as YouTube doesn't let its content be reached easily.

Another workaround solution apart from OBS is to just use the AMS sample WebRTC publish page and share the YouTube video tab. In this way, you can send the stream to your AMS easily.

Thank you
